src/db/tables/featureFlags.ts

```diff
@@ -38,17 +38,10 @@ export async function getDeletedFeatureFlags() {
 }
 
 export const featureFlagNames = [
-  "UpdatedEmailPreferencesOption",
   "CancellationFlow",
-  "ConfirmCancellation",
-  "FirstDataBrokerRemovalFixedEmail",
   "DiscountCouponNextThreeMonths",
   "LatestScanDateCsatSurvey",
-  "AutomaticRemovalCsatSurvey",
-  "AdditionalRemovalStatuses",
```

Comment:
I believe this is also a feature flag we would need to keep. We’ve [recently disabled it](https://mozilla.slack.com/archives/C0460QALW69/p1736794513970349?thread_ts=1736544869.685789&cid=C0460QALW69) with an issue related to the last opt-out removal. Even though, this issue will be addressed by PR #5591 the flag `AdditionalRemovalStatuses`, it depends on the opt-out removal to be activated and to work correctly.

Reply:
Reverted this in 82df2e2e1. Could you create a ticket that specifies when this flag is good to remove?